### DISCUSSION TITLE
How to create a installable setup for Linux

### DISCUSSION THREAD

**Discussion by SithuBoy** [unoplatform/uno#10764] [Q&A]
I need to know how to create a setup for ubuntu from skia.gtk of uno platform. Debugging is working but i cannot figure out how to make setup for linux.

**Comment by jeromelaban** [comment]
Thanks for the question. You can build [a standalone folder](https://learn.microsoft.com/en-us/dotnet/core/deploying/), like this, but it will be a manual installation.

We do not yet have documentation about it, but you can use AppImage [here](https://github.com/unoplatform/calculator/blob/uno/build/appimage/AppImageBuilder.x64.yaml) for the Uno Calculator, and you can also package your app using [Snap](https://github.com/unoplatform/calculator/tree/uno/build/snap).
